<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForm</class>
 <widget class="QDockWidget" name="SearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Поиск</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название местности</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="textGeo">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:4;
border-width:1px;
max-height:20px;
min-height:20px;

}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название сайта</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="textURL">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:4;
border-width:1px;
max-height:20px;
min-height:20px;

}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Формат</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
border-radius:4;
border-width:1px;
max-height:20px;
min-height:20px;

}
QComboBox:editable{

background-color: rgb(255, 255, 255);
border-radius:4}</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Shapefile</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>jpg</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="btnSearch">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>127</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>106</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>56</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>127</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>106</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>56</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>127</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>106</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>56</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>30</red>
            <green>30</green>
            <blue>30</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.022598900000000" starty="0.023000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="ReflectSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.000000000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.813559000000000">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </gradientstop>
            <gradientstop position="1.000000000000000">
             <color alpha="255">
              <red>162</red>
              <green>162</green>
              <blue>184</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:qlineargradient(spread:reflect, x1:0.0225989, y1:0.023, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.813559 rgba(204, 204, 204, 255), stop:1 rgba(162, 162, 184, 255));
font: 9pt ;
	border-color: rgb(144, 144, 144);
	
	color: rgb(30, 30, 30);
border-width: 1px;
border-radius:4;
border-style:solid;

min-width:150px;
max-width:150px;
min-height:20px;
max-height:20px;
}
</string>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btnSearch</zorder>
   <zorder>label</zorder>
   <zorder>textGeo</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
